src/test/features/ccj/routes/paid-amount-summary.ts

```diff
@@ -0,0 +1,53 @@
+import { expect } from 'chai'
+import * as request from 'supertest'
+import * as config from 'config'
+
+import { attachDefaultHooks } from '../../../routes/hooks'
+import '../../../routes/expectations'
+
+import { Paths } from 'ccj/paths'
+
+import { app } from '../../../../main/app'
+
+import * as idamServiceMock from '../../../http-mocks/idam'
+import * as claimStoreServiceMock from '../../../http-mocks/claim-store'
+import * as draftStoreServiceMock from '../../../http-mocks/draft-store'
+import { checkAuthorizationGuards } from './checks/authorization-check'
+import { sampleClaimObj } from '../../../http-mocks/claim-store'
+
+const cookieName: string = config.get<string>('session.cookieName')
+const externalId = sampleClaimObj.externalId
+const paidAmountSummaryPage = Paths.paidAmountSummaryPage.uri.replace(':externalId', externalId)
+
+describe('CCJ - paid amount summary page', () => {
+  attachDefaultHooks()
+
+  describe('on GET', () => {
+    checkAuthorizationGuards(app, 'get', paidAmountSummaryPage)
+
+    context('when user authorised', () => {
+      beforeEach(() => {
+        idamServiceMock.resolveRetrieveUserFor(1, 'cmc-private-beta')
+      })
+
+      it('should return 500 and render error page when cannot retrieve claims', async () => {
```

Comment:
I would be also interested to see the behaviour when:
- retrieving a CCJ draft failed
- that page is not accessible when claim is not eligible for CCJ e.g. `checkAlreadySubmittedGuard(app, 'get', ResponsePaths.defendantAddressPage.uri)` done for response

Reply:
there's a test for the guard specifically, I didn't want to go running that test on every view, can do if needed but it seemed overkill..

Reply:
@timja-kainos can you point me to the guard test? It might be enough if it makes sure that guard is applied to each CCJ path. If it is not we might miss the issue where guard has not been properly applied to some CCJ paths.

Reply:
https://github.com/hmcts/cmc-citizen-frontend/blob/master/src/test/features/ccj/routes/ccjGuard.ts

Reply:
added test case for "retrieving a CCJ draft failed"

Reply:
I think having guard tests in separate file is fine but it is testing just one particular route at the moment. I would prefer to have tests for all eligible routes. Maybe could be done by iterating through list of routes in `ccj/paths`

Reply:
added tests for guard src/test/features/ccj/routes/ccjGuard.ts